examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -75,11 +64,10 @@ def download_file(url, file):
 # We will load a sample from the LibriSpeech test-other dataset.
 # 
 
-speech_url = "https://pytorch.s3.amazonaws.com/torchaudio/tutorial-assets/ctc-decoding/8461-258277-0000.flac"
-speech_file = "_assets/speech.flac"
+speech_url = "https://pytorch.s3.amazonaws.com/torchaudio/tutorial-assets/ctc-decoding/8461-258277-0000.wav"
+speech_file = "/tmp/speech.wav"
```

Comment:
Can you use [`torch.hub.get_dir`](https://pytorch.org/docs/stable/hub.html#torch.hub.get_dir)? So that configurable default temporary directory will be selected.

Hardcoding `/tmp` would leave artifacts when running locally.